testing/rclone/APKBUILD

```diff
@@ -0,0 +1,98 @@
+# Contributor: tcely <rclone+aports@tcely.33mail.com>
+# Maintainer: tcely <rclone+aports@tcely.33mail.com>
+_pkgname="rclone"
+pkgname="${_pkgname}"
+pkgver="1.48.0"
+pkgrel=0
+pkgdesc="rsync for cloud storage"
+url="https://rclone.org/"
+arch="all"
+license="MIT"
+options=""
+depends=""
+makedepends="bash git go perl"
+install=""
+subpackages=""
+
+gpg_signature_extensions="."
+gpgfingerprints="good:FBF7 37EC E9F8 AB18 604B  D2AC 9393 5E02 FF3B 54FA"
+gpgsource="${pkgname}-${pkgver}-sha256sums::https://downloads.rclone.org/v${pkgver}/SHA256SUMS"
+source="https://downloads.rclone.org/v${pkgver}/${_pkgname}-v${pkgver}.tar.gz $gpgsource"
+
+builddir="${srcdir}/${_pkgname}-v${pkgver}"
+export GOPATH="${srcdir}/go"
+
+sanitycheck() {
```

Comment:
This is very strange.
Perhaps this functionality should wait until after abuild has these features? (I assume this is done to allow for gpg checks)

Reply:
This is done to allow for verifying before any sums are calculated.

Reply:
You're disabling native sanity checks in order to allow for something custom built into the file that's meant to describe how to build the package.
Saying that you're doing it to allow said custom thing isn't really a resolution of any kind.

Reply:
I've disabled selected sanity checks for source and sums counting because they blocked the verification step from running.

Reply:
So... back to the original question.
Perhaps this functionality should wait until after abuild has these features, and to focus efforts on giving abuild native support for gpg?

You're actively *DISABLING* security features to add in unproven and untested code.

Reply:
What security features are you referring to?

Reply:
By setting source to null, you are disabling [this entire block](https://github.com/alpinelinux/abuild/blob/master/abuild.in#L202).
It does the following things:
1. Makes sure `$install` files are not in `$source`
2. Checks if wget is in makedepends for https sources.
3. Checks to make sure that all source entries have SOME hash attached to them (!).
4. Checks for potentially invalid tarball/source names.

It also disables [this part](https://github.com/alpinelinux/abuild/blob/master/abuild.in#L230) which makes sure that there are no outdated checksums (it's common to manually add a new checksum without removing old ones, while source is emptied, which could introduce problems).

These are not replaced at any point.
Further, even if they were, why would this be done in the APKBUILD rather than as a part of the abuild? (again, the source question, that remains unanswered).

Reply:
LGTM with new changes. Feel free to resolve 👍